lib/ecto/adapters/clickhouse/structure.ex

```diff
@@ -67,6 +65,9 @@ defmodule Ecto.Adapters.ClickHouse.Structure do
       rows = rows |> IO.iodata_to_binary() |> String.replace("),(", "),\n(")
       versions = ["INSERT INTO ", table, " (version, inserted_at) VALUES\n", rows, ";\n"]
       {:ok, versions, conn}
+    else
+      {:ok, _, conn} ->
+        {:ok, "", conn}
```

Comment:
Why is this needed? When is the result not `%{rows: rows}`?

Reply:
I could take this out, this is kind of a niche case. I am migrating from a different Clickhouse client so I want to create a structure dump. Because it's not Ecto, there is no `schema_migrations` table at all. I manually created the table, but it doesn't contain any records yet. In this case, the dump command produces `INSERT INTO schema_migrations VALUES ;`

Reply:
I see. But I don't think this change actually fixes the issue. We'd need to add a check on rows being non-empty for this  `else` clause to be matched.

Reply:
You are right, I made a mistake with committing. I changed it to what I meant.